### PR TITLE
CRNS-118/Indicators

### PR DESCRIPTION
**Pull request by DanC5**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

## Description of the pull request

Indicator components, Vish said these don't need tests

### CODE REVIEW

File: src/components/Indicators/LoadingIndicator.js [hunk 1]
```diff
@@ -1,65 +1,55 @@
...
   ${({ theme }) => theme.loadingIndicator.loadingText.css}
 `;
 
-class LoadingIndicator extends React.PureComponent {
-  static propTypes = {
-    listType: PropTypes.oneOf(['channel', 'message', 'default']),
-    loadingText: PropTypes.string,
-  };
+const LoadingIndicator = (props) => {
+  const { listType, loadingText } = props;
```
Comment (virdesai) [inline src/components/Indicators/LoadingIndicator.js:22]: are we using a spread of props somewhere else or should we just destructure this in the method param?

File: src/components/Indicators/LoadingIndicator.js [hunk 2]
```diff
@@ -1,65 +1,55 @@
...
+};
+
+LoadingIndicator.propTypes = {
+  listType: PropTypes.oneOf(['channel', 'message', 'default']),
+  loadingText: PropTypes.string,
+};
+
+LoadingIndicator.defaultProps = {
+  listType: 'default',
+};
```
Comment (virdesai) [inline src/components/Indicators/LoadingIndicator.js:53]: if you destructure this in the method param then you can set `listType = 'default'` within that

Comment (DanC5) [reply src/components/Indicators/LoadingIndicator.js]: updated

**Comment by vishalnarkhede** [comment]
Lets change the base branch to master!!

**Comment by DanC5** [comment]
> Lets change the base branch to master!!

updated!

**Pull request merged**